### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ramonsmits**
Based on feedback by @adamralph 

@mikeminutillo @tmasternak Might we just past the whole first section of the tutorial in the readme instead of linking to it?

### CODE REVIEW

**Comment by ramonsmits** [comment]
@adamralph That is why I'm asking @tmasternak and @mikeminutillo if we would want to copy the first section of the tutorial in the README as that contains what happens.

https://github.com/Particular/docs.particular.net/blob/monitoring-gettingstarted2/tutorials/monitoring/demo/index.md#running-the-sample

Would that help?

**Comment by adamralph** [comment]
@ramonsmits I see. Yes, I think that would help. Currently it's not at all obvious that I should follow that link to find what running run.bat will do to my machine. The two things are in different parts of the document.

**Comment by tmasternak** [comment]
@adamralph can you have a look at current version. I've added single sentence per each option that explicitly states what the script will do.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,36 @@
-# MonitoringDemo
-Self-contained demo showing all of the monitoring components working together. 
+# Overview
+
+Self-contained demo showing all of the monitoring components working together. This accompanies the Particular Platform Monitoring Tutorial available at the following location:
+
+- https://docs.particular.net/tutorials/monitoring/demo/
```
Comment (adamralph) [inline README.md]: This URL is a 404.

Comment (tmasternak) [reply README.md]: Yes this is 404 until we merge the tutorial:/ (which should be today or tomorrow)

**Comment by tmasternak** [comment]
> Does the script do enough to call out any other pre-requisities?

Yes if you have a fresh Windows installation pre-requisites are all you need to have the demo running.

**Comment by tmasternak** [comment]
@adamralph I'm merging this PR. We can polish the `Readme.md` further in next releases.

**Pull request merged**